linchpin/provision/roles/openstack/tasks/provision_resource_group.yml

```diff
@@ -25,13 +25,16 @@
 
 - name: "Get creds from auth driver"
   auth_driver:
-    filename: "{{ cred_filename }}" # don't include the .yaml
+    filename: "{{ cred_filename }}"
```

Comment:
Why remove the useful comment? Did the .yaml get added back to the cred_filename?